Steps/PublishCodeCoverage.yml

```diff
@@ -0,0 +1,45 @@
+## @file
+# Azure Pipelines step template to merge and publish all code coverage results.
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+parameters:
+- name: include_paths
+  displayName: Tool Chain (e.g. VS2022)
+  type: string
+  default: ''
+- name: exclude_paths
+  displayName: Tool Chain (e.g. VS2022)
+  type: string
+  default: ''

```

Comment:
`include_paths` and `exclude_paths` are not used in this file. Also the `displayName` comment seems unrelated.

Reply:
Yes, I decided to drop this in one of the stages and forgot to remove the parameters